helm-charts/charts/kube-starrocks/charts/operator/templates/clusterrolebinding.yaml

```diff
@@ -11,4 +11,9 @@ subjects:
 - kind: ServiceAccount
   name: {{ template  "operator.serviceAccountName" . }}
   namespace: {{ template  "operator.namespace" . }}
-{{- end }}
\ No newline at end of file
+{{- if .Values.global.rbac.serviceAccountName }}
+- kind: ServiceAccount
+  name: {{ .Values.global.rbac.serviceAccountName }}
```

Comment:
why not make the `operator.serviceAccountName` to global.rbac.serviceAccountName, if user provides it?

Reply:
In `global.rbac`, there was a `global.rbac.create` which controls whether to create service account.